6.5/cdm/m-cluster/am/global/Realms/root.json

```diff
@@ -11,6 +11,6 @@
     "parentPath" : null,
     "active" : true,
     "name" : "/",
-    "aliases" : [ "&{fqdn}", "data", "openam", "login.&{namespace}.svc.cluster.local"]
+    "aliases" : [ "&{fqdn}", "data", "openam", "openam.&{namespace}.svc.cluster.local"]
```

Comment:
Have you tried just http://openam

cluster.local is not always going to be the same on every cluster

Reply:
yes, IG needs the full fqdn as I tried with just openam.  I could always parameterize the internal domain so it can be changed if necessary.